I1724 ibm style guide (#1221)
codenvy/che-docs: src/main/pages/che-7/overview/ref_che-7-known-issues.adoc

== Permission denied errors when installing {prod} in the default namespace

> When installing {prod} in the `default` namespace, the proper security contexts and file permissions are not set appropriately, causing the installation to fail. It is recommended for users to avoid installation of {prod-short} in the `default` namespace and instead create a new namespace, for example, `{project-context}`, to contain the {prod-short} installation.

Reviewer: ```suggestion
When installing {prod} in the `default` namespace, the proper security contexts and file permissions are not set appropriately, causing the installation to fail. Recommendation: do not install {prod-short} in the `default` namespace. Instead, create a new namespace, for example, `{project-context}`, to contain the {prod-short} installation.
```
